guides/examples/Amazon SQS.md

```diff
@@ -1,3 +1,185 @@
 # Amazon SQS
 
-TODO.
\ No newline at end of file
+Amazon Simple Queue Service (SQS) is a highly scalable distributed message
+queuing service provided by Amazon.com. AWS SQS offers two types of message
+queues:
+
+  * Standard
+    * Nearly unlimited throughput
+    * Best-effort ordering
+    * At-least-once delivery
+
+  * FIFO
+    * Limited number of transactions per second (TPS).
+      See AWS SQS documentation for more information on limits.
+    * Order in which messages are sent/received is strictly preserved 
+    * Exactly-once delivery
+  
+Broadway can work seamlessly with both, Standard and FIFO queues.
+
+## Getting Started
+
+In order to use Broadway with SQS, we basically need to:
```

Comment:
```suggestion
In order to use Broadway with SQS, we need to:
```

It is generally recommended to avoid words like "just", "only", "basically", etc. :)